sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1711,7 +1828,26 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 
 			case 234:
 				currentVersion = 234;
-				TableUtils.createTable(connectionSource, SormasToSormasOriginInfo.class);
+
+				getDao(SormasToSormasOriginInfo.class).executeRaw(
+					"CREATE TABLE sormasToSormasOriginInfo ("
+					+ "		comment VARCHAR,"
+					+ "		organizationId VARCHAR,"
```

Comment:
I think we did not have an organizationId here at the time of this commit.
Instead there was a string field healthDepartment

https://github.com/hzi-braunschweig/SORMAS-Project/commit/67d6e446f79263fbf59f6315db47702593d86bd7

Reply:
https://github.com/hzi-braunschweig/SORMAS-Project/blob/38465a1a7fea70f4b830784514fb7e46843235c7/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sormastosormas/SormasToSormasOriginInfo.java#L20-L25

I found `organizationId` in version 1.50.0

See [SormasToSormasOriginInfo.java Line 23](https://github.com/hzi-braunschweig/SORMAS-Project/blob/38465a1a7fea70f4b830784514fb7e46843235c7/sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sormastosormas/SormasToSormasOriginInfo.java#L20-L25)

Reply:
@MartinWahnschaffe I hope you understand my response.

During the implementation of this fix,
I went through the tagged branches to extract all change points and their app version.

I then downloaded the releases of affected versions
and installed them one after the other.
For each installation, I debug and extract the sqlite db file.
I then inspect affected tables and extract their schema into raw sql statements.

In short, I didn't approach it the way you do with commit inspection